rust/tools/authorization-logic/BUILD

```diff
@@ -0,0 +1,98 @@
+#-------------------------------------------------------------------------------
+# Copyright 2021 Google LLC
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+#
+#-------------------------------------------------------------------------------
+package(default_visibility = ["//visibility:public"])
+
+load("@rules_rust//rust:rust.bzl", "rust_binary", "rust_library")
+
+licenses(["notice"])
+
+filegroup(
+    name = "antlr4rust_jar",
+    srcs = ["third_party/antlr4-rust/antlr4rust.jar"]
+)
+
+filegroup(
+    name = "antlr_gen_mod_rs",
+    srcs = ["src/parsing/antlr_gen/mod.rs"]
+)
+
+# Generate the parsing-related files using antlr4rust
+genrule(
+    name = "antlr_gen_srcs",
+    srcs = ["src/parsing/AuthLogic.g4"],
+    tools = [
+        ":antlr4rust_jar",
+	":antlr_gen_mod_rs",
```

Comment:
Nit: One of these two lines is indented incorrectly.

Reply:
Done.